.github/actions/setup/action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
   cache:
     description: "Cache Maven repo (true/false/restore)"
     default: true
+  cleanup-node:
+    description: "Clean up node (true/false) to increase free disk space"
+    default: false
```

Comment:
Why false by default?

Add a comment or enable by default.

test-other-modules was recently failing due to out of disk space, so i guess many jobs could benefit

Reply:
This adds 2 minutes to every job, which is a significant overhead. I suggested only enabling this for jobs that we know to require more disk space.